R/load_data.R

```diff
@@ -0,0 +1,106 @@
+# Helper functions not for export
+
+#' Check for a version of data to load
+#'
+#' This function determines if a user is requesting the latest version
+#' of the bundled data or wants to access one of the earlier veresions.
+#'
+#' @param mode Determines which data to handle. Defaults to somatic_hypermutation_locations. Will grow with more options as more data is version tracked.
+#' @param this_genome_build The genome build of the data if coordinate-based. Accepts grch37 (default) or hg38.
+#'
+#' @return data frame
+#' @import config dplyr readr GAMBLR.data
+#'
```

Comment:
I have gone over all the GAMBLR helper functions on my branch. This was done as a step in preparing the documentation for building a site from the source code with `build_site` from `pkgdown`. This function takes all `.Rd` files that live in the man folder and generates function-specific HTML files. This is not something we want for helper functions (since they are not exported in NAMESPACE). 

The fix for this is to add `@noRd` in the documentation for such functions. This prevents the `.Rd` file from being generated. 

I think you should also specify this for this function to keep things consistent. Let me know if this doesn't make sense, or if you need me to further clarify. 

Thanks,